src/content/docs/vulnerability-management/integrations/aws.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: AWS security integration
+metaDescription: 'Send your security data from AWS Security Hub, GuardyDuty, and inspector directly to New Relic.'
```

Comment:
Should GuardyDuty be GuardDuty? I'm actually not sure if it's something I've never heard of so just wanted to check.